From 44f20382cc03643c8fb287dd243ccd1ebdfef89e Mon Sep 17 00:00:00 2001
From: Chien Wong <m@xv97.com>
Date: Sat, 27 Apr 2024 16:19:43 +0800
Subject: [PATCH] NAN: Fix a typo in USD doc

The 'req_instance' parameter in transmit command should be
'req_instance_id'.

Fixes: e3f9ab3c3a01 ("NAN: USD in wpa_supplicant")
Signed-off-by: Chien Wong <m@xv97.com>
---
 wpa_supplicant/README-NAN-USD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wpa_supplicant/README-NAN-USD b/wpa_supplicant/README-NAN-USD
index 5dfe6eeed..72c379fc9 100644
--- a/wpa_supplicant/README-NAN-USD
+++ b/wpa_supplicant/README-NAN-USD
@@ -75,7 +75,7 @@ NAN_CANCEL_SUBSCRIBE subscribe_id=<id from NAN_SUBSCRIBE>
 
 This command maps to the CancelSubscribe() method in the NAN Discovery Engine.
 
-NAN_TRANSMIT handle=<id from NAN_PUBLISH or NAN_SUBSCRIBE> req_instance=<peer's id> address=<peer's MAC address> [ssi=<service specific information (hexdump)>]
+NAN_TRANSMIT handle=<id from NAN_PUBLISH or NAN_SUBSCRIBE> req_instance_id=<peer's id> address=<peer's MAC address> [ssi=<service specific information (hexdump)>]
 
 This command maps to the Transmit() method in the NAN Discovery Engine.
 
-- 
2.43.0

